src/core/types.ts

```diff
@@ -69,7 +69,7 @@ export interface SceneObject<TState extends SceneObjectState = SceneObjectState>
   readonly urlSync?: SceneObjectUrlSyncHandler<TState>;
 
   /** Subscribe to state changes */
-  subscribeToState(observer?: Partial<Observer<TState>>): Subscription;
+  subscribeToState(observer?: Partial<Observer<TState>>): SubscriptionLike;
```

Comment:
Is there a use case for subscribe without an observer?
```suggestion
  subscribeToState(observer: Partial<Observer<TState>>): SubscriptionLike;
```